services/model.js

```diff
@@ -293,19 +289,49 @@
               that.saving = true
               return that.sync(that.getIdentifier() ? 'PUT' : 'POST',
                 that.toJSON({
-                  patch: true,
+                  patch: true
                 })).catch(function (message) {
                 $mdToast.show(
-                  $mdToast.simple().
-                    textContent('Failure to Save!').
-                    toastClass('errorMessage').
-                    position('top right').
-                    hideDelay(3000)
+                  $mdToast.simple()
+                    .textContent('Failure to Save!')
+                    .toastClass('errorMessage')
+                    .position('top right')
+                    .hideDelay(3000)
                 )
                 $log.error('SAVE:', message)
               })
             }
 
+            /**
+             * @returns {*}
+             */
+            this.iterateVersion = function () {
```

Comment:
This is too specific and will need to be edited throughout the growth of Sitetheory.  If you want to make this more abstract, you may be able to continue.